actionpack/test/controller/force_ssl_test.rb

```diff
@@ -92,6 +92,22 @@ def cheeseburger
   end
 end
 
+class RedirectToSSLIfSessionStoreDisabled < ForceSSLController
```

Comment:
Build [failed](https://travis-ci.org/rails/rails/jobs/178503726). But we can ignore that.

> PG::TRDeadlockDetected: ERROR:  deadlock detected